text.md

```diff
@@ -0,0 +1 @@
+This file  should be ignored by git
```

Comment:
- Kindly check why this file was not ignored by git as requested on the project requirement. You need to follow all the requirements.